src/reporter/delete.py

```diff
@@ -46,6 +65,7 @@ def delete_entities(entity_type, from_date=None, to_date=None,
                                         fiware_service=fiware_s(),
                                         fiware_servicepath=fiware_sp(),)
 
+
         logging.getLogger(__name__).info(
```

Comment:
after deleting the block above (lines 55 - 67) we should make sure this statement and all the following ones are properly indented, i.e. indentation should start at the same level as the statement

    deleted = trans.delete_entities(....

on line 50?